src/fireedge/src/modules/containers/Users/Users.js

```diff
@@ -0,0 +1,214 @@
+/**
+ * Displays a list of Users with a split pane between the list and selected row(s).
+ *
+ * @returns {ReactElement} Users list and selected row(s)
+ */
+export function Users() {
+  const [selectedRows, setSelectedRows] = useState(() => [])
+  const actions = UsersTable.Actions()
+  const { zone } = useGeneral()
+
+  return (
+    <TranslateProvider>
+      <ResourcesBackButton
+        selectedRows={selectedRows}
+        setSelectedRows={setSelectedRows}
+        useUpdateMutation={UserAPI.useUpdateUserMutation}
+        zone={zone}
+        actions={actions}
+        table={(props) => (
+          <UsersTable.Table
+            onSelectedRowsChange={props.setSelectedRows}
+            globalActions={props.actions}
+            onRowClick={props.resourcesBackButtonClick}
+            useUpdateMutation={props.useUpdateMutation}
+            zoneId={props.zone}
+            initialState={{
+              selectedRowIds: props.selectedRowsTable,
+            }}
+          />
+        )}
+        simpleGroupsTags={(props) => (
+          <GroupedTags
+            tags={props.selectedRows}
+            handleElement={props.handleElement}
+            onDelete={props.handleUnselectRow}
+          />
+        )}
+        info={(props) => {
+          const propsInfo = {
+            user: props?.selectedRows?.[0]?.original,
+            selectedRows: props?.selectedRows,
+          }
+          props?.gotoPage && (propsInfo.gotoPage = props.gotoPage)
+          props?.unselect && (propsInfo.unselect = props.unselect)
+
+          return <InfoTabs {...propsInfo} />
+        }}
+      />
+    </TranslateProvider>
+  )
+}
+
+/**
+ * Displays details of an User.
+ *
+ * @param {User} user - User to display
+ * @param {Function} [gotoPage] - Function to navigate to a page of an User
+ * @param {Function} [unselect] - Function to unselect a User
+ * @returns {ReactElement} User details
+ */
+const InfoTabs = memo(({ user, gotoPage, unselect }) => {
+  const [get, { data: lazyData, isFetching }] = UserAPI.useLazyGetUserQuery()
+  const id = user?.ID ?? lazyData?.ID
+
+  const { settings: { FIREEDGE: fireedge = {} } = {} } = useAuth()
+  const { FULL_SCREEN_INFO } = fireedge
+  const { fullViewMode } = SERVER_CONFIG
+  const fullModeDefault =
+    FULL_SCREEN_INFO !== undefined ? FULL_SCREEN_INFO === 'true' : fullViewMode
+  const { isFullMode } = useGeneral()
+  const { setFullMode } = useGeneralApi()
+
+  useEffect(() => {
+    !isFullMode && gotoPage()
+  }, [])
+
+  return (
+    <Stack overflow="auto">
+      <Stack
+        direction="row"
+        alignItems="center"
+        justifyContent="space-between"
+        gap={1}
+        mx={1}
+        mb={1}
+      >
+        <Stack direction="row">
+          {fullModeDefault && (
+            <SubmitButton
+              data-cy="detail-back"
+              icon={<NavArrowLeft />}
+              tooltip={Tr(T.Back)}
+              isSubmitting={isFetching}
+              onClick={() => unselect()}
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Using Function.prototype.bind and arrows functions as attributes will negatively impact performance in React. Each time the parent is rendered, the function will be re-created and trigger a render of the component causing excessive renders and more memory use.